srv/components/uds/teardown.sls

```diff
@@ -30,6 +35,10 @@ Delete uds yum repo:
   pkgrepo.absent:
     - name: {{ defaults.uds.repo.id }}
 
+Remove uds service file:
+  file.absent:
+    - name: /usr/lib/systemd/system/uds.service
```

Comment:
Is there any customization in the service file?
If not, this cleanup should happen during `yum erase uds`

Reply:
No customization as such, we're copying these service files from our `salt://files` . 
Some `systemd` files got removed with teardown and some did not. (ex, `uds.service`) 
So, as a failsafe option, added this line for all service files.